src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/ExceptionManagement/VolleyExceptionFilterAttribute.cs

```diff
@@ -5,10 +5,12 @@
     using System.Web.Routing;
     using Crosscutting.Contracts.Infrastructure;
 
+#pragma warning disable S3993 // Custom attributes should be marked with "System.AttributeUsageAttribute"
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)